SPECS/kernel/config

```diff
@@ -7353,6 +7345,7 @@ CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y
 CONFIG_LOCK_DOWN_KERNEL_FORCE_NONE=y
 # CONFIG_LOCK_DOWN_KERNEL_FORCE_INTEGRITY is not set
 # CONFIG_LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY is not set
+CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT=y
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/5af5f7ab7e82ae36e9501b5b642d16d96294a2b1#diff-e3e2a2e6e4a51a1110ee88068d24ca30aea5b4c17da145761d455e687a51a51d